components/delegations/DelegateCard.tsx

```diff
@@ -77,76 +79,74 @@ export default function DelegateCard({ delegate }: PropTypes): React.ReactElemen
-            width: bpi > 1 ? 'auto' : '100%',
-            marginTop: bpi > 1 ? 0 : 4
+            width: ['100%', '100%', 'auto', '100%', 'auto'],
+            mt: [4, 4, 0, 4, 0]
           }}
         >
           <Grid
             columns={[2]}
             sx={{
-              width: bpi > 1 ? 'auto' : '100%',
-              marginBottom: bpi > 1 ? 0 : 4
+              width: ['100%', '100%', 'auto', '100%', 'auto'],
+              mb: [4, 4, 0, 4, 0]
             }}
           >
-            <Box sx={{ mr: [4] }}>
+            <Flex sx={{ flexDirection: 'column', justifyContent: 'space-between' }}>
               <Box sx={{ mb: 3 }}>
                 <Text as="p" variant="microHeading" sx={{ fontSize: [3, 5] }}>
                   {totalStaked ? totalStaked.toBigNumber().toFormat(2) : '0.00'}
                 </Text>
-                <Text as="p" variant="secondary" color="onSecondary">
+                <Text as="p" variant="secondary" color="onSecondary" sx={{ fontSize: ['13px', '15px'] }}>
                   Total MKR delegated
                 </Text>
               </Box>
               <Box>
                 <Text as="p" variant="microHeading" sx={{ fontSize: [3, 5], color: 'secondaryMuted' }}>
                   Untracked
                 </Text>
-                <Text as="p" variant="secondary" color="onSecondary">
+                <Text as="p" variant="secondary" color="onSecondary" sx={{ fontSize: ['13px', '15px'] }}>
                   Pool participation
                 </Text>
               </Box>
-            </Box>
+            </Flex>
 
-            <Box sx={{ mr: [0, 0, 4] }}>
+            <Box sx={{ mr: [0, 0, 4, 0, 4] }}>
               <Box sx={{ mb: 3 }}>
                 <Text as="p" variant="microHeading" sx={{ fontSize: [3, 5] }}>
                   {mkrStaked ? mkrStaked.toBigNumber().toFormat(2) : '0.00'}
                 </Text>
-                <Text as="p" variant="secondary" color="onSecondary">
+                <Text as="p" variant="secondary" color="onSecondary" sx={{ fontSize: ['13px', '15px'] }}>
                   MKR delegated by you
                 </Text>
               </Box>
               <Box>
                 <Text as="p" variant="microHeading" sx={{ fontSize: [3, 5], color: 'secondaryMuted' }}>
                   Untracked
                 </Text>
-                <Text as="p" variant="secondary" color="onSecondary">
+                <Text as="p" variant="secondary" color="onSecondary" sx={{ fontSize: ['13px', '15px'] }}>
                   Executive participation
                 </Text>
               </Box>
             </Box>
           </Grid>
 
-          <Box
+          <Flex
             sx={{
-              textAlign: 'right',
-              width: ['100%', '100%', 'auto'],
-              display: ['flex', 'flex', 'block'],
+              width: ['100%', '100%', 'auto', '100%', 'auto'],
+              flexDirection: ['row', 'row', 'column', 'row', 'column'],
```

Comment:
Do we really have 5 breakpoints to account for? That seems excessive, maybe we could simplify and target 3 (mobile, tablet, desktop)?

Reply:
Yeah. I don't even think tablet is meaningful these days anymore. If we optimize for phone and desktop we are covering already the most important things.
![image](https://user-images.githubusercontent.com/1152768/126146560-98c9e8c8-353e-4df7-b40b-b84d95b261be.png)

But this is something we can cover on another conversation

Reply:
The breakpoints are part of the theme UI spec. So we could discuss if want to change this going forward.  

The challenge here is the sidebar layout. The width of the workable space on the page grows, then shrinks again when the sidebar appears, then continues to grow again.

Reply:
Let's redo the layout in the future at some point 🚀